### ISSUE TITLE
[BUG] Reading order is confusing

### ISSUE THREAD

**Issue by livcheerful** [livcheerful/anywhereadventures#318]
**Describe the bug**
Timestamp - 2:07

**To Reproduce**
1. Turn on Voiceover (if iPhone)
3. Open reading page from map popup
4. Tab through to pane by swiping right
6. Order goes "previous story" -> text of body -> share button -> reads correctly

**Expected behavior**
It should be: 
"previous story" -> "next story" -> top level card? maybe we shouldn't go through all button options (see in log, show on map, copy address, share) or have a way to jump to content? -> content

**Screenshots**
For reference:

<img width="346" height="567" alt="Image" src="https://github.com/user-attachments/assets/6c3ba07c-6feb-4521-8551-f65ebb1acd7a" />
